packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-granules.js

```diff
@@ -119,6 +120,12 @@ test.beforeEach(async (t) => {
   );
 });
 
+test.afterEach.always(async (t) => {
+  await t.context.knex(tableNames.files).del();
```

Comment:
Generally speaking I prefer that tests clean up exactly what they created rather than wiping tables between tests, but it's not a hill that I will die on

Reply:
I'm happy to fight about it 😄 . What I like about the `afterEach nuke the entire table` approach is that you're guaranteed a fresh slate. I do hear ya though. If you decide to feel strongly about it, I'm happy to relent.

Reply:
The thing i don't like about that is that then you don't know exactly what the state is created by the code under test or the test setup itself. So your tests can be passing even the logic under test is leaving around some state that you don't know about/expect